common/vm_get_ip.yml

```diff
@@ -2,30 +2,42 @@
 # SPDX-License-Identifier: BSD-2-Clause
 ---
 # Get VM IP Address using esxcli command or from VMware Tools guestinfo
-- name: Initialize VM guest IP address variable
+# Parameter:
+#   vm_primary_nic_mac: The VM's primary network adapter MAC address must be set
+# Return:
+#   vm_guest_ip: The VM's guest IPv4 address
+#
+- name: "Check vm_primary_nic_mac is set"
```

Comment:
This check also need to be added in tasks `vm_get_ip_from_notools.yml`, `vm_get_ip_from_vmtools.yml` in case they are called separately.

Reply:
Parameter checking was added in the two sub task files. But it is not recommended to call them directly as vm_get_ip.yml would check VM's power state and power on VM if necessary, the two sub tasks didn't check power state.